cmd/scollector/collectors/google_cloudbilling.go

```diff
@@ -0,0 +1,234 @@
+package collectors
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"golang.org/x/oauth2/google"
```

Comment:
organize these better. We usually use three blocks: std lib, third party, bosun.